assignments/objects.js

```diff
@@ -18,28 +18,80 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const interns = [
+  {
+  "id": 0,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+},
+{
+"id": 1,
+"name": "Kennan",
+"email": "kdiben1@tinypic.com",
+"gender": "M",
+speak() {
+  return `Hello, my name is ${this.name}!`;
+}
+},
+{
+"id": 2,
+"name": "Keven",
+"email": "kmummery2@wikimedia.org",
+"gender": "M"
+},
+{
+"id": 3,
+"name": "Gannie",
+"email": "gmartinson3@illinois.edu",
+"gender": "M"
+},
+{
+"id": 4,
+"name": "Aintonietta",
+"email": "adaine5@samsung.com",
+"gender": "F",
+multiplyNums: function(x, y){
+  return x*y;
+}
+}
+]
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(interns[0].name);
+
 // Kennan's ID
 
+console.log(interns[1].id);
+
 // Keven's email
 
+console.log(interns[2].email);
+
 // Gannie's name
 
+console.log(interns[3].name);
+
 // Antonietta's Gender
 
+console.log(interns[4].gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+console.log(kennan.speak())
+
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+console.log(antonietta.multiplyNums(3,4));
```

Comment:
If you define which objects `kennan` and `antonietta` are, the corresponding console logs will work.